FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -7,18 +7,28 @@ import Foundation
 public final class RemoteFeedLoader: FeedLoader {
 	private let url: URL
 	private let client: HTTPClient
+	private let mapper: FeedImageMapper
 
 	public enum Error: Swift.Error {
 		case connectivity
 		case invalidData
 	}
 
-	public init(url: URL, client: HTTPClient) {
+	public init(url: URL, client: HTTPClient, mapper: FeedImageMapper = RemoteFeedImageMapper.default) {
 		self.url = url
 		self.client = client
+		self.mapper = mapper
 	}
 
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
-		fatalError("Must be implemented")
+		client.get(from: url) { [weak self] result in
+			guard let self = self else { return }
+			switch result {
+			case .success((let data, let response)):
```

Comment:
The `let` can be defined outside the tuple so the expression can be shorter:

```suggestion
			case let .success((data, response)):
```